collection-spec/collection-spec.md

```diff
@@ -90,10 +90,10 @@ and consumers have not been granted any explicit right to use the data.
 
 Collections are *strongly recommended* to provide summaries of the values of fields that they can expect from the `properties` 
 of STAC Items contained in this Collection. This enables users to get a good sense of what the ranges and potential values of 
-different fields in the Collection are, without to inspect a number of Items (or crawl them exhaustively to get a definitive answer). 
+different fields in the Collection are, without having to inspect a number of Items (or crawl them exhaustively to get a definitive answer). 
 Summaries help to fully define Collections, especially if they don't link to any Items. They also give clients enough information to 
```

Comment:
I originally found this reference to `fully define Collections` confusing since if there are no Items referenced in the collection the summary would be empty from the definition above. Can an Item be `contained` in a Collection but not referenced?

Reply:
Yeah, that line is confusing. I don't think an item can be in a collection but not referenced. This is more meant to describe Standalone Collections, that use summaries to give an overview of their data, but don't actually link to items. 

```suggestion
Summaries are often used to give users a sense of the data in [Standalone Collections](#standalone-collections), describing the potential values even when it can't be accessed as Items. They also give clients enough information to 
```